recipes/setuptools/bld.bat

```diff
@@ -0,0 +1,8 @@
+python setup.py install
```

Comment:
Why not run `bootstrap.py`? This should be included in the package as well. It might avoid these weird issues of `setuptools` needing `setuptools`.